sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -863,17 +863,10 @@ void notify(@NonNull Error error,
             callback.beforeNotify(report);
         }
 
-        HandledState handledState = report.getError().getHandledState();
-
-        if (handledState.isUnhandled()) {
-            sessionTracker.incrementUnhandledError();
-        } else {
-            sessionTracker.incrementHandledError();
-            if (sessionTracker.getCurrentSession() != null) {
-                setChanged();
-                notifyObservers(new NativeInterface.Message(
-                            NativeInterface.MessageType.NOTIFY_HANDLED, error.getExceptionName()));
-            }
+        if (sessionTracker.getCurrentSession() != null) {
+            setChanged();
+            notifyObservers(new NativeInterface.Message(
```

Comment:
previously we would only notify observers for handled errors (presumably because of unpredictable behaviour after an unhandled error?)

Reply:
I've updated this to only notify if the error object is `handled` and has a session.